docs/paths_doc.md

```diff
@@ -83,6 +83,33 @@ Returns `True` if `path` is an absolute path.
 `True` if `path` is an absolute path.
 
 
+<a id="paths.is_normalized"></a>
+
+## paths.is_normalized
+
+<pre>
+paths.is_normalized(<a href="#paths.is_normalized-str">str</a>, <a href="#paths.is_normalized-look_for_same_level_references">look_for_same_level_references</a>)
+</pre>
+
+Returns true if the passed path contains uplevel references "..".
```

Comment:
Are the docs autogenerated from code? If not, this and the RETURNS section below need changing.

Reply:
Regenerated.